sensorlib/src/objects/DetectedObject.py

```diff
@@ -45,19 +45,24 @@ def build_detected_object(carla_actor, allowed_semantic_tags):
         if (bounding_box == None):
             return None
         
+        projection_string = projection_string_config
+
+        size_x = carla_actor.bounding_box.extent.x*2
```

Comment:
Why are we multiplying by 2 here?

Reply:
Can we remove the 2 here. I think we are not sure about this calculation so lets not do any arbitrary multiplication